openscap_report/cli.py

```diff
@@ -1,3 +1,6 @@
+# Copyright 2022, Red Hat, Inc.
+# SPDX-License-Identifier: LGPL-2.1
```

Comment:
GPL would be a better fit, as this is the script that calls the rest of the product that is a LGPL-licensed module.